src/api/app/services/meta_controller_service/project_updater.rb

```diff
@@ -0,0 +1,30 @@
+module MetaControllerService
+  class ProjectUpdater
+    def initialize(project: nil, request_data: {}, validator_klass: ::MetaControllerService::RequestValidator)
+      @project = project
+      @request_data = request_data
+      @validator = validator_klass.new(project: project, request_data: request_data)
+    end
+
+    def call
+      @validator.call
```

Comment:
I would prefer to get rid of this additional ``call`` call. I think this should behave similar to ActiveRecord which also does not require an additional method call before calling ``valid?``. See my other comment.